ASTProcessor/src/test/java/edu/usb/argos/ASTProcessor/visitor/JavaMethodVisitorTest.java

```diff
@@ -0,0 +1,218 @@
+package edu.usb.argos.ASTProcessor.visitor;
+
+import edu.usb.argos.ASTProcessor.antlr.JavaLexer;
+import edu.usb.argos.ASTProcessor.antlr.JavaParser;
+import edu.usb.argos.ASTProcessor.visitor.core.entities.method.*;
```

Comment:
As I said before about the asterisks * it would be good if you didn't call everyone, do the same with the others.